static/js/vue-cdr-access/tests/unit/abstract.spec.js

```diff
@@ -0,0 +1,182 @@
+import { mount } from '@vue/test-utils'
+import { createRouter, createWebHistory } from 'vue-router';
+import abstract from '@/components/full_record/abstract.vue';
+import displayWrapper from '@/components/displayWrapper.vue';
+import {createI18n} from 'vue-i18n';
+import translations from '@/translations';
+import cloneDeep from 'lodash.clonedeep';
+
+const recordData = {
+    pageSubtitle: 'testAdminUnit',
+    briefObject: {
```

Comment:
the structure of this object is out of date since we realize we were returning the briefObject in the wrong form, but i don't think it'll impact what you're doing too much. For your tests, all I think you need is:
```
{
    abstractText: 'Lorem ipsum dolor sit amet',
    id: '9e3b1754-8e39-494d-8cc4-44e0ddf264d9'
}
```